website/source/docs/builders/amazon.html.md

```diff
@@ -202,7 +202,10 @@ work, but specifics will depend on your use-case.
 {
     "Sid": "PackerIAMPassRole",
```

Comment:
Could you update ```$id``` to?

Reply:
Can you clarify what you mean here?

Reply:
That statemenet is no longer only about ```PassRole```, but also for ```GetInstanceProfile```, so ```$id``` as  ```PackerIAMPassRole``` no longer match.

Reply:
Ah, I see. Something more generic does make sense now.